test_data/generated/resolve/bpftrace.json

```diff
@@ -18,5 +18,91 @@
       "name": "toplevel",
       "page": null
     }
+  ],
+  [
+    {
+      "msgs": [],
+      "name": "toplevel",
+      "page": {
+        "complete": true,
+        "msgs": [],
+        "packages": [
+          {
+            "attr_path": "bpftrace",
+            "broken": false,
+            "derivation": "/nix/store/iskf0w9ijnpswyd6jh1yc0aba164alj4-bpftrace-0.21.2.drv",
+            "description": "High-level tracing language for Linux eBPF",
+            "insecure": false,
+            "install_id": "bpftrace",
+            "license": "Apache-2.0",
+            "locked_url": "https://github.com/flox/nixpkgs?rev=99dc8785f6a0adac95f5e2ab05cc2e1bf666d172",
+            "name": "bpftrace-0.21.2",
+            "outputs": [
+              {
+                "name": "man",
+                "store_path": "/nix/store/7x4y4vdjdqq5vw6m4v06k41wrwsvckqp-bpftrace-0.21.2-man"
+              },
+              {
+                "name": "out",
+                "store_path": "/nix/store/nsxhqzrcpy1r3a0f0z06bjy3mkck5vyc-bpftrace-0.21.2"
+              }
+            ],
+            "outputs_to_install": [
+              "out",
+              "man"
+            ],
+            "pname": "bpftrace",
+            "rev": "99dc8785f6a0adac95f5e2ab05cc2e1bf666d172",
```

Comment:
suggestion (blocking to make a decision): this introduces another rev. I think we should probably regenerate all mocks whenever we add a file? That causes a lot of churn but I don't think there's really much reason not to? It will also keep us in better sync with prod

Reply:
I think if we implement regular mock regens it would be clearer to only (re) generate the required mock in a PR and have the revs synched with the next periodical regen.
besides being clearer/more focused, regenning all mocks makes it more likely that unrelated tests start failing